### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by thisisamanmishra** [community/community#59373] [GitHub Education]
### Hi everyone,

I want to apply for Student Developer Pack but when I select my College Name then,  it shows that the I must have my college issued email id but my college doesn't provide email id with his domain name and it shows that it is compulsory, What should I do now?

**Comment by samus-aran** [comment]
Hi @thisisamanmishra  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

It would also be recommended that either you or a faculty member provides evidence that students do not receive academic emails to help with the request. 👍

Hope this helps! Let us know how you get along 👍
